site-src/concepts/guidelines.md

```diff
@@ -118,6 +140,16 @@ The "implementation-specific" designation allows a CORE or EXTENDED feature to
 be well-defined taking into account the realities of some features that are
 mostly but not entirely portable.
 
+## Limitations of Webhook and CRD Validation
+CRD and webhook validation is not the final validation i.e. webhook is “nice UX”
+but not schema enforcement. This validation is intended to provide immediate
+feedback to users when they provide an invalid configuration, but can not
+completely be relied on because it:
```

Comment:
"Relied on to (what?) because it:"

It's not obvious to me from this what it shouldn't be relied on for?

Maybe the "DO NOT rely on webhook or CRD validation" text from above would fit better in this section?

Reply:
Good points. Updated this section, hopefully it's a bit more clear now.